### DISCUSSION TITLE
Background img not displaying on Github pages

### DISCUSSION THREAD

**Discussion by jolinshouse** [community/community#127784] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I deployed a page to Github pages and it displays everything except the background image. I think there is no problem in the path. Please help.

Here is the repo: https://github.com/jolinshouse/jolinshouse.github.io.git
(The url for the background image is in the html file)

Github page: https://jolinshouse.github.io/

**Comment by davevad93** [comment]
Hi @jolinshouse ,

I guess the issue is with `Cover.JPG` because in the browser's dev console returns a 404 error:

![0](https://github.com/community/community/assets/94308234/b5232284-6a19-4952-b19b-da70ce6a9468)

I think the problem is here in your `index.html`:

```html
<header class="header" style="height: 504px; background-image: url('./images/cover.JPG');">
``` 

GitHub Pages is case sensitive so change `cover.JPG` to `Cover.JPG`, that should fix the issue.

**Reply by jolinshouse** [reply, marked as answer]
It's worked!! Thank you👍

**Reply by davevad93** [reply]
You're welcome @jolinshouse , by the way your website looks awesome! 👍
